Sources/SentrySwiftUI/SentryTracedView.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 import Sentry
-#if SWIFT_PACKAGE || CARTHAGE
-import SentryInternal
+#if CARTHAGE || SWIFT_PACKAGE
```

Comment:
@brustolin `import SentryInternal` will work with SPM, since there is additional lib target `SentryInternal`: https://github.com/getsentry/sentry-cocoa/blob/1ee3d54a9529f3d2343600e8af1442c00f42b3cf/Package.swift#L31-L37

Reply:
It works. But we dont need to expose it, so using `@_implementationOnly` its fine